string frame_id
bool frame_locked
float32 yaw
---
bool success
string message
